src/apps/settings/src/app/views/Fonts/Browse.js

```diff
@@ -215,7 +222,7 @@ export default connect(state => {
                 <h3 className={styles.FontFamily}>{itemFont.family}</h3>
                 <ul className={styles.FontVariants}>
                   {itemFont.variants.map((item, index) => (
-                    <li key={`${itemFont.family}-${index}`}>
+                    <li key={`${itemFont.family}-${item}`}>
```

Comment:
What is `item`? Is this something we should be adding into a component key? i.e. is this a string?